Src/ILGPU.Algorithms.Tests/Configurations.txt

```diff
@@ -1,6 +1,6 @@
 ﻿ArrayExtensionTests: Debug, Release, O2
 GroupExtensionTests: Debug, Release, O2
-﻿HistogramTests: Debug, Release, O2
+// ﻿HistogramTests: Debug, Release, O2
```

Comment:
ISSUE: Restored Histogram tests.

Reply:
👍 I also just noticed when running the test that the histogram tests were not being executed.